### ISSUE TITLE
deliver download_metadata: [!] undefined method `+' for nil:NilClass

### ISSUE THREAD

**Issue by fguchelaar** [fastlane/fastlane#16949]
### New Issue Checklist

- [x] Updated fastlane to the latest version (running 2.153.1)
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
When downloading the metadata using deliver, it gives an error: `` [!] undefined method `+' for nil:NilClass ``.

The App has multiple locales set in AppStore Connect. They all seem to be downloaded okay, but after reading the general data (review_information), it crashes.

##### Command executed
`fastlane deliver download_metadata`

##### Complete output when running fastlane, including the stack trace and command used
<details>
<summary>Output summary</summary>

### deliver 2.153.1 Summary
| Key | Value |
| --- | ---|
| screenshots_path                     | ./fastlane/screenshots           |
| metadata_path                        | ./fastlane/metadata              |
| app_version                          | 1.4.0                            |
| username                             | *****             |
| app_identifier                       | ***** |
| ipa                                  | *****.ipa         |
| platform                             | ios                              |
| edit_live                            | false                            |
| use_live_version                     | false                            |
| skip_binary_upload                   | false                            |
| skip_screenshots                     | false                            |
| skip_metadata                        | false                            |
| skip_app_version_update              | false                            |
| force                                | false                            |
| overwrite_screenshots                | false                            |
| submit_for_review                    | false                            |
| reject_if_possible                   | false                            |
| automatic_release                    | false                            |
| phased_release                       | false                            |
| reset_ratings                        | false                            |
| team_id                              | *****                        |
| run_precheck_before_submit           | true                             |
| precheck_default_rule_level          | warn                             |
| ignore_language_directory_validation | false                            |
| precheck_include_in_app_purchases    | true                             |

### Stacktrace

```
Looking for related GitHub issues on fastlane/fastlane...

➡️  fastlane env crashes with undefined method `strip' for nil:NilClass
    https://github.com/fastlane/fastlane/issues/16748 [open] 1 💬
    3 weeks ago

➡️  [Swift] undefined method `each' for nil:NilClass - Cannot Create Group Within FastlaneSwiftRunner Project
    https://github.com/fastlane/fastlane/issues/15184 [open] 37 💬
    2 weeks ago

➡️  [!] undefined method `[]' for #<Gem::NameTuple fastlane, 2.143.0, ruby> (NoMethodError)
    https://github.com/fastlane/fastlane/issues/16127 [open] 23 💬
    a week ago

and 445 more at: https://github.com/fastlane/fastlane/search?q=undefined%20method%20%60%2B%27%20for%20nil&type=Issues&utf8=✓

🔗  You can ⌘ + double-click on links to open them directly in your browser.
Traceback (most recent call last):
	14: from /usr/local/bin/fastlane:23:in `<main>'
	13: from /usr/local/bin/fastlane:23:in `load'
	12: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/bin/fastlane:23:in `<top (required)>'
	11: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/fastlane/lib/fastlane/cli_tools_distributor.rb:111:in `take_off'
	10: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/commands_generator.rb:17:in `start'
	 9: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/commands_generator.rb:186:in `run'
	 8: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
	 7: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
	 6: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
	 5: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
	 4: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
	 3: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/commands_generator.rb:180:in `block (2 levels) in run'
	 2: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/setup.rb:131:in `generate_metadata_files'
	 1: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/setup.rb:131:in `each'
/Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/setup.rb:137:in `block in generate_metadata_files': undefined method `+' for nil:NilClass (NoMethodError)
	14: from /usr/local/bin/fastlane:23:in `<main>'
	13: from /usr/local/bin/fastlane:23:in `load'
	12: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/bin/fastlane:23:in `<top (required)>'
	11: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/fastlane/lib/fastlane/cli_tools_distributor.rb:111:in `take_off'
	10: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/commands_generator.rb:17:in `start'
	 9: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/commands_generator.rb:186:in `run'
	 8: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
	 7: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
	 6: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
	 5: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
	 4: from /Library/Ruby/Gems/2.6.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
	 3: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/commands_generator.rb:180:in `block (2 levels) in run'
	 2: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/setup.rb:131:in `generate_metadata_files'
	 1: from /Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/setup.rb:131:in `each'
/Library/Ruby/Gems/2.6.0/gems/fastlane-2.153.1/deliver/lib/deliver/setup.rb:137:in `block in generate_metadata_files': [!] undefined method `+' for nil:NilClass (NoMethodError)
```
</details>

### Environment

<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                                          |
| --------------------------- | -------------------------------------------------------------- |
| OS                          | 10.15.6                                                        |
| Ruby                        | 2.6.3                                                          |
| Bundler?                    | false                                                          |
| Git                         | git version 2.23.0                                             |
| Installation Source         | /usr/local/bin/fastlane                                        |
| Host                        | Mac OS X 10.15.6 (19G73)                                       |
| Ruby Lib Dir                | /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib |
| OpenSSL Version             | LibreSSL 2.8.3                                                 |
| Is contained                | false                                                          |
| Is homebrew                 | false                                                          |
| Is installed via Fabric.app | false                                                          |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                    |
| Xcode Version               | 11.6                                                           |

### System Locale

| Error                       |
| --------------------------- |
| No Locale with UTF8 found 🚫 |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
fastlane_version "2.8.0"

default_platform :ios

platform :ios do
 
  desc "Push a new beta build to TestFlight"
  lane :beta do

    changelog = prompt(
      text: "Changelog: ",
      multi_line_end_keyword: "END"
    )
    
    build_app
    upload_to_testflight(changelog: changelog)
  end
end

```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier "*****" # The bundle identifier of your app
apple_id "*****" # Your Apple email address

itc_team_id "*****"  # Developer Portal Team ID
```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.153.1 | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.4.0        |
| slack-notifier            | 2.3.2        |
| atomos                    | 0.1.3        |
| claide                    | 1.0.3        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.6        |
| xcodeproj                 | 1.15.0       |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.3.0        |
| terminal-notifier         | 2.0.0        |
| unicode-display_width     | 1.7.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.5.0        |
| addressable               | 2.7.0        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.7.1        |
| tty-cursor                | 0.7.1        |
| tty-spinner               | 0.9.3        |
| babosa                    | 1.0.3        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.6        |
| excon                     | 0.73.0       |
| unf_ext                   | 0.0.7.7      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20190701 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday                   | 1.0.1        |
| faraday_middleware        | 1.0.0        |
| fastimage                 | 2.1.7        |
| gh_inspector              | 1.1.3        |
| mini_magick               | 4.10.1       |
| rubyzip                   | 2.3.0        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.7.5        |
| naturally                 | 2.2.0        |
| simctl                    | 1.6.8        |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.2        |
| mini_mime                 | 1.0.2        |
| multi_json                | 1.14.1       |
| signet                    | 0.14.0       |
| memoist                   | 0.16.2       |
| os                        | 1.1.0        |
| googleauth                | 0.11.0       |
| httpclient                | 2.8.3        |
| google-api-client         | 0.37.3       |
| google-cloud-env          | 1.3.1        |
| google-cloud-errors       | 1.0.0        |
| google-cloud-core         | 1.5.0        |
| digest-crc                | 0.5.1        |
| google-cloud-storage      | 1.26.0       |
| jmespath                  | 1.4.0        |
| aws-partitions            | 1.295.0      |
| aws-eventstream           | 1.0.3        |
| aws-sigv4                 | 1.1.1        |
| aws-sdk-core              | 3.93.0       |
| aws-sdk-kms               | 1.30.0       |
| aws-sdk-s3                | 1.61.2       |
| CFPropertyList            | 3.0.2        |
| public_suffix             | 4.0.4        |
| json                      | 2.3.0        |
| bundler                   | 2.1.4        |
| jwt                       | 2.2.1        |
| emoji_regex               | 2.0.0        |
| forwardable               | 1.3.1        |
| logger                    | 1.4.2        |
| stringio                  | 0.1.0        |
| ipaddr                    | 1.2.2        |
| openssl                   | 2.1.2        |
| ostruct                   | 0.1.0        |
| strscan                   | 1.0.3        |
| date                      | 3.0.0        |
| fileutils                 | 1.4.1        |
| io-console                | 0.5.6        |
| zlib                      | 1.1.0        |
| libxml-ruby               | 3.1.0        |
| rexml                     | 3.2.4        |
| psych                     | 3.1.0        |
| mutex_m                   | 0.1.0        |
| webrick                   | 1.6.0        |
</details>

**Comment by SebaRossi94** [comment]
Hello there, i have the same problem here with a few differences:
- Only one locale set (spanish MX)
- Command executed was `fastlane deliver init`
- Crash happens when downloading `review_information/review_demo_password.txt`

Here it is the complete output:

```
Traceback (most recent call last):
        16: from /usr/local/bin/fastlane:23:in `<main>'
        15: from /usr/local/bin/fastlane:23:in `load'
        14: from /var/lib/gems/2.5.0/gems/fastlane-2.153.1/bin/fastlane:23:in `<top (required)>'
        13: from /var/lib/gems/2.5.0/gems/fastlane-2.153.1/fastlane/lib/fastlane/cli_tools_distributor.rb:111:in `take_off'
        12: from /var/lib/gems/2.5.0/gems/fastlane-2.153.1/deliver/lib/deliver/commands_generator.rb:17:in `start'
        11: from /var/lib/gems/2.5.0/gems/fastlane-2.153.1/deliver/lib/deliver/commands_generator.rb:186:in `run'
        10: from /var/lib/gems/2.5.0/gems/commander-fastlane-4.4.6/lib/commander/delegates.rb:15:in `run!'
         9: from /var/lib/gems/2.5.0/gems/fastlane-2.153.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:76:in `run!'
         8: from /var/lib/gems/2.5.0/gems/commander-fastlane-4.4.6/lib/commander/runner.rb:476:in `run_active_command'
         7: from /var/lib/gems/2.5.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:153:in `run'
         6: from /var/lib/gems/2.5.0/gems/commander-fastlane-4.4.6/lib/commander/command.rb:178:in `call'
         5: from /var/lib/gems/2.5.0/gems/fastlane-2.153.1/deliver/lib/deliver/commands_generator.rb:118:in `block (2 levels) in run'
         4: from /var/lib/gems/2.5.0/gems/fastlane-2.153.1/deliver/lib/deliver/setup.rb:21:in `run'
         3: from /var/lib/gems/2.5.0/gems/fastlane-2.153.1/deliver/lib/deliver/setup.rb:49:in `generate_deliver_file'
         2: from /var/lib/gems/2.5.0/gems/fastlane-2.153.1/deliver/lib/deliver/setup.rb:131:in `generate_metadata_files'
         1: from /var/lib/gems/2.5.0/gems/fastlane-2.153.1/deliver/lib/deliver/setup.rb:131:in `each'
/var/lib/gems/2.5.0/gems/fastlane-2.153.1/deliver/lib/deliver/setup.rb:137:in `block in generate_metadata_files': [!] undefined method `+' for nil:NilClass (NoMethodError)
```
that specific field doesn't have any special characters only numbers and letters

**Comment by fguchelaar** [comment]
✅ Verified the fix with 2.154.0. Thanks!
